src/app/components/courses-page/courses/courses.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+import { ICourse } from 'src/app/shared/interfaces/course';
+
+@Component({
+  selector: 'app-courses',
+  templateUrl: './courses.component.html',
+  styleUrls: ['./courses.component.scss'],
+})
+export class CoursesComponent {
+  @Input() coursesList: ICourse[] = [];
+  @Output() deleteCouseItem = new EventEmitter<string | number>();
```

Comment:
Please add type for EventEmitter - it is good practice to add type if the type is not primitive

Reply:
Done